examples/security/v_fia/SplitGuard/slmodel_splitguard.py

```diff
@@ -0,0 +1,486 @@
+class SLModel_SG(SLModel):
+    def __init__(
+        self,
+        base_model_dict: Dict[Device, Callable[[], "tensorflow.keras.Model"]] = {},
+        device_y: PYU = None,
+        model_fuse: Callable[[], "tensorflow.keras.Model"] = None,
+        dp_strategy_dict: Dict[Device, DPStrategy] = None,
+        random_seed: int = None,
+        backend: str = "tensorflow",
+        strategy="split_nn",
+        agg_method: AggMethod = None,
+        N: int = 20,
+        alpha: float = 1.0,
+        beta: int = 1,
+        fake_batch: List = [],
+        **kwargs,
+    ):
+        """Interface for vertical split learning
+        Attributes:
+            base_model_dict: Basemodel dictionary, key is PYU, value is the Basemodel defined by party.
+            device_y: Define which model have label.
+            model_fuse:  Fuse model definition.
+            dp_strategy_dict: Dp strategy dictionary.
+            random_seed: If specified, the initial value of the model will remain the same, which ensures reproducible.
+            backend: name of backend engine, tensorflow or torch, default tensorflow
+            strategy: Strategy of split learning.
+            agg_method: agg method decide how to aggregate hiddens from each parties, default None(compatible with legacy mode)
+        Keyword Args
+            simulation: Only need when use simulation mode
+            base_local_steps: Only for 'split_async' strategy, Number of rounds for local base update process
+            fuse_local_steps: Only for 'split_async' strategy, Number of rounds for local fuse update process
+            bound_param: Only for 'split_async' strategy, Parameter for limiting local gradient change
+            loss_thres: Only for 'split_state_async' strategy, Loss threshold triggering switch state in splitStateAS strategy
+            split_steps: Only for 'split_state_async' strategy, Number of batches triggering switch state in splitStateAS strategy
+            max_fuse_local_steps: Only for 'split_state_async' strategy, Maximum number of rounds for fuse local update in splitStateAS strategy?
+            compressor: Define strategy tensor compression algorithms to speed up transmission.
+            device_agg: The party do aggregation,it can be a PYU,SPU,etc.
+        """
+        super().__init__(
+            base_model_dict,
+            device_y,
+            model_fuse,
+            dp_strategy_dict,
+            random_seed,
+            backend,
+            strategy,
+            agg_method,
+            **kwargs,
+        )
+        self.alhpa = alpha
+        self.beta = beta
+        self.N = N
+        self.fake_batch = fake_batch
+        # Add data to compute SG Score
+        self.fake, self.r_1, self.r_2, self.results, self.fake_indices = (
+            [],
+            [],
+            [],
+            [],
+            [],
+        )
+
+    # Add function to convert data format to ndarray.
```

Comment:
Stick to [google style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) please.
Function docstring should like 
```
def convert_to_ndarray:
"""
# Add function to convert data format to ndarray.
"""

Reply:
And it seems that this comment is meaningless since the function name is explicit enough.